xskillscore/core/deterministic.py

```diff
@@ -1070,11 +1093,6 @@ def mape(a, b, dim=None, weights=None, skipna=False, keep_attrs=False):
     xarray.Dataset or xarray.DataArray
         Mean Absolute Percentage Error.
 
-    See Also
-    --------
-    xarray.apply_ufunc
```

Comment:
scikit-learn recently had MAPE added (https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/_regression.py#L197) but I don't think it's been released yet so I don't think we can link it to the docs (or update our testing against it)